composer.json

```diff
@@ -24,7 +24,7 @@
     "phpunit/phpunit": "^9.6",
     "wp-phpunit/wp-phpunit": "dev-master",
     "yoast/phpunit-polyfills": "^1.0@dev",
-    "brain/monkey": "^2.0"
+    "brain/monkey": "^2.0@dev"
```

Comment:
There's no need to include the `composer.json` and `composer.lock` files in this PR. Could you please revert those changes?

Reply:
I reverted the composer.json to its previous state, but unfortunately, I lost the composer.lock file in the process. I had to generate a new one using composer update to ensure the dependencies were correctly locked in place.